lib/carrierwave/mongoid.rb

```diff
@@ -4,6 +4,7 @@
 require 'mongoid-grid_fs'
 require 'carrierwave'
 require 'carrierwave/validations/active_model'
+require 'byebug'
```

Comment:
But this shows that you **did not** fully remove it before pushing. It's why the entire build is failing on Travis at the moment.